examples/Web/KafkaClientHandle.cs

```diff
@@ -0,0 +1,41 @@
+using Confluent.Kafka;
+using Microsoft.Extensions.Configuration;
+
+
+namespace Web
+{
+    /// <summary>
+    ///     Wraps a Confluent.Kafka.IProducer instance, and allows for basic
+    ///     configuration of this via IConfiguration.
+    ///    
+    ///     KafkaClientHandle does not provide any way for messages to be produced
+    ///     directly. Instead, it is a dependency of KafkaDependentProducer. You
+    ///     can create more than one instance of KafkaDependentProducer (with
+    ///     possibly differing K and V generic types) that leverages the same
+    ///     underlying producer instance exposed by the Handle property of this
+    ///     class. This is more efficient than creating separate
```

Comment:
Add "vastly", "greatly", or some other word to describe the humungous difference in efficiency between producer-per-message vs singleton

Reply:
i'm actually just talking about different types here. I'll add another note about what you're getting at.